frontend/Makefile

```diff
@@ -0,0 +1,7 @@
+# use bash as the default shell to be used in the recipe
```

Comment:
It's worth looking into Bazel at some point. https://github.com/bazelbuild/rules_go is a good starting point.